test/registration_spec.rb

```diff
@@ -58,6 +60,10 @@
       expect(available_addons.find { |addon| addon.identifier == "sle-sdk" }).to \
         receive(:registered)
 
+      # the received product renames are passed to the software management
+      expect(Registration::SwMgmt).to receive(:update_product_renames)
+        .with("SUSE_SLES_SAP" => "SLES_SAP")
+
       allow(File).to receive(:exist?).with(
```

Comment:
NTH: This is something I'll also comment in the other pull request I'm reviewing right now. I would have expected a different ```it``` to test this.

In this module in general we abuse of huge ```it```s with tons of ```expect``` instead of smaller and more specific ``it```s with only a few ```expect```, using ```allow``` when we need to mock something.

Since this PR is more urgent, I will LGTMed even with this concern. But I will be more strict in the other PRs so we fix it for the future. :-)